### PR TITLE
Add a Python example alongside the cURL example in the Aura API docs

### PR DESCRIPTION
**Pull request by fiquick**
Start with just an example on how to get the auth token in Python

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-324.surge.sh

Updated pages:
- [Authentication](https://neo4j-docs-aura-324.surge.sh/aura/platform/api/authentication)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-324.surge.sh

Updated pages:
- [Authentication](https://neo4j-docs-aura-324.surge.sh/aura/platform/api/authentication)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-324.surge.sh

Updated pages:
- [Authentication](https://neo4j-docs-aura-324.surge.sh/aura/platform/api/authentication)

File: modules/ROOT/pages/platform/api/authentication.adoc [hunk 1]
```diff
@@ -74,7 +74,12 @@ Both the request and response contain sensitive information and must be kept sec
 You are responsible for keeping the client credentials and access tokens confidential, whether in transit (by specifying HTTPS), if stored at rest, in log files, etc.
 ====
 
-==== cURL example
+==== Code examples
```
Comment (nvitucci) [inline modules/ROOT/pages/platform/api/authentication.adoc:77]:
```suggestion
==== Request examples 
```

File: modules/ROOT/pages/platform/api/authentication.adoc [hunk 2]
```diff
@@ -86,6 +91,33 @@ curl --request POST 'https://api.neo4j.io/oauth/token' \
...
+    "POST",
+    "https://api.neo4j.io/oauth/token",
+    headers={"Content-Type": "application/x-www-form-urlencoded"},
+    data={"grant_type": "client_credentials"},
+    auth=HTTPBasicAuth(client_id, client_secret),
+    timeout=10
+)
+
+# Obtain the token to use with our Aura API from the response
+token = response.json()["access_token"]
```
Comment (nvitucci) [inline modules/ROOT/pages/platform/api/authentication.adoc:114]:
```suggestion
print(response.json())
```

I would say let's just print the whole response in JSON format, so we are consistent with the output from cURL.

**Review by nvitucci — CHANGES_REQUESTED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-324.surge.sh

Updated pages:
- [Authentication](https://neo4j-docs-aura-324.surge.sh/aura/platform/api/authentication)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-324.surge.sh

Updated pages:
- [Authentication](https://neo4j-docs-aura-324.surge.sh/aura/platform/api/authentication)

**Review by nvitucci — APPROVED** [review]

**Pull request merged**